docs/src/token.md

```diff
@@ -449,6 +458,17 @@ consistently valid Multisig accounts all Solana accounts holding a Account,
 Mint, or Multisig must contain enough SOL to be considered [rent
 exempt](https://docs.solana.com/implemented-proposals/rent)
 
```

Comment:
Let's put this under a new subsection, and then link to it from "the known issue" above:
`#### Rent-exemption loophole`